src/tailwind.js

```diff
@@ -106,6 +113,9 @@ module.exports = {
           600: "#ffbf00",
         },
       },
+      boxShadow: {
+        dropdown: `0px 4px 20px rgba(0, 0, 0, 0.15)`,
+      },
```

Comment:
use tw de? i more for maintenance rather than pixel perfect 😬

Reply:
jerry rejected the tw style :x hahhaha